airbyte-integrations/connectors/destination-mysql/src/test/java/io/airbyte/integrations/destination/mysql/MySQLDestinationTest.java

```diff
@@ -0,0 +1,81 @@
+package io.airbyte.integrations.destination.mysql;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+import static org.mockito.Mockito.spy;
+
+import com.fasterxml.jackson.databind.JsonNode;
+import com.google.common.collect.ImmutableMap;
+import io.airbyte.commons.json.Jsons;
+import java.util.Map.Entry;
+import org.junit.jupiter.api.Test;
+
+public class MySQLDestinationTest {
```

Comment:
Can you add a unit test case in which the config as `ssl=false` and those SSL related parameters won't be appended? I think you wrote one previously.

Reply:
done